go.sum

```diff
@@ -22,6 +22,8 @@ github.com/stretchr/testify v1.7.0 h1:nwc3DEeHmmLAfoZucVR881uASk0Mfjw8xYJ99tb5Cc
 github.com/stretchr/testify v1.7.0/go.mod h1:6Fq8oRcR53rry900zMqJjRRixrwX3KX962/h/Wwjteg=
 github.com/xdg-go/stringprep v1.0.2 h1:6iq84/ryjjeRmMJwxutI51F2GIPlP5BfTvXHeYjyhBc=
 github.com/xdg-go/stringprep v1.0.2/go.mod h1:8F9zXuvzgwmyT5DUm4GUfZGDdT3W+LCvS6+da4O5kxM=
+go.mongodb.org/atlas v0.9.1-0.20210707095538-d4a51579e8e0 h1:bcBUWQgNiq2WYnM+R7vUKIGvZKuKIc3b0Pcn/t1nYw8=
+go.mongodb.org/atlas v0.9.1-0.20210707095538-d4a51579e8e0/go.mod h1:MMWDsc2akjTDSG4tVQrxv/82p3QbBnqeELbtTl45sbg=
```

Comment:
`go mod tidy`?

Reply:
I have noticed that if I run  `go mod tidy`,` go.mongodb.org/atlas` will point to the latest release v0.10 which is quite behind.

Reply:
but v0.10 > 0.9.1, this is why tidy prefers that, I think someone messed up the versions on atlas, can you check that?

Reply:
[This](https://github.com/mongodb/go-client-mongodb-atlas/blob/d4a51579e8e0c1caec75a14f612bdc6a1dd6e213/.github_changelog_generator#L2) is the only place where I see v9.0

Reply:
I'm not sure why `go get -u go.mongodb.org/atlas@master` is tagging as 0.9.1 but doing that and then mod tidy is correctly cleaning up the 0.10.0 version, so please run `go mod tidy` here

Reply:
That is not possible because of some fixes that were merged in the atlas go client after the 0.10.0 for the invitation management endpoints so we have to make another release for that.

Reply:
Done

Reply:
I am not sure why it does not work for me

Reply:
in order to make it "work" I have to run `go get -u go.mongodb.org/atlas@v0.10.0` but if we use the 0.10.0 here, e2e tests wills start to fail on mongocli because this version does not have some fixes in the invite management.

Reply:
I have checked the commits missed in v0.10.0 and we may be okay with it, I thought that I have changed one struct in one of those commits but I was wrong. I am going to use v.0.10.0 here

Reply:
updated

Reply:
updated

Reply:
not sure what it's on your side but like I said it works for me https://github.com/mongodb/go-client-mongodb-ops-manager/pull/112/files